back/sssscs/src/main/java/com/ib/user/User.java

```diff
@@ -19,12 +25,48 @@
 @AllArgsConstructor
 @Entity
 @Table(name = "users") // Must not be "user".
-public class User {
-	@Id 
+public class User implements UserDetails {
+
+	public enum Role {
+		FOO,
```

Comment:
We should add the actual roles right now, there's only two roles anyway (not counting the logged-out user).

Reply:
https://github.com/magley/sssscs/pull/2/commits/d58052c8b2394e9fdccabc86620a4121b5472ab4